front/app/component-library/components/Input/index.tsx

```diff
@@ -204,6 +207,22 @@ class Input extends PureComponent<InputProps> {
           required={required}
           autoComplete={autocomplete}
           onKeyDown={onKeyDown}
+          onPaste={
```

Comment:
Could be cleaner to define on top instead of in the JSX.

Reply:
Hmm but then I lose automatic type interference of the event type... :(

Reply:
You can always add the type. In any case, not a blocker.